app/controllers/actions/AuthAction.scala

```diff
@@ -97,7 +99,8 @@ class AuthActionImpl @Inject()(override val authConnector: AuthConnector,
       case emcsEnrolments =>
         emcsEnrolments.find(_.identifiers.exists(ident => ident.key == EnrolmentKeys.ERN && ident.value == ernFromUrl)) match {
           case Some(enrolment) if enrolment.isActivated =>
-            block(UserRequest(request, ernFromUrl, internalId, credId, sessionId.get.value, emcsEnrolments.size > 1))
+            //            block(UserRequest(request, ernFromUrl, internalId, credId, sessionId.get.value, emcsEnrolments.size > 1))
```

Comment:
Probably need to remove the commented line?

Reply:
Addressed in https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/57/commits/f2d17b54feea4abda070e4d59e4a8b420c3e0a92